components/delivery-option.jsx

```diff
@@ -31,37 +31,41 @@ export function DeliveryOption ({
 		<div
 			id="deliveryOptionField"
 			className={divClassName}
+			role="group"
+			aria-label="Delivery options"
 		>
-			{
-				options.map((option, index) => {
-					const inputProps = {
-						type: 'radio',
-						id: option.value,
-						name: 'deliveryOption',
-						value: option.value,
-						className: 'o-forms__radio o-forms__radio--right ncf__delivery-option__input',
-						...(option.isSelected && { defaultChecked: true })
-					};
+			<span className="o-forms-input o-forms-input--radio-round">
+				{
+					options.map((option, index) => {
+						const inputProps = {
+							type: 'radio',
+							id: option.value,
+							name: 'deliveryOption',
+							value: option.value,
+							className: 'ncf__delivery-option__input',
+							...(option.isSelected && { defaultChecked: true })
```

Comment:
If this is only ever going to be a boolean, could you just do `isSelected: option.isSelected`?

Reply:
Shouldn't be just...
```suggestion
							defaultChecked: option.isSelected
```
?